src/index.ts

```diff
@@ -83,6 +84,8 @@ function compareSpecificity(xpathStringA: string, xpathStringB: string): -1 | 0
  */
 const domFacade = new ExternalDomFacade() as IDomFacade;
 
+declare var window;
```

Comment:
Seems work around for strict mode issue. Let's make a proper fix when we enable strict mode.

Reply:
No, turns out that lib.dom also defines window. Because we dropped that library, and the window access is optional (part of a UMD wrapper), the declare is needed.

Reply:
Can we add a comment describing the reason we need this? Same as with the other declaration.